src/native/fs/fs_napi.cpp

```diff
@@ -379,11 +379,10 @@ struct Writefile : public FSWorker
         auto buf = info[2].As<Napi::Buffer<uint8_t>>();
         _data = buf.Data();
         _len = buf.Length();
-        Begin(XSTR() << DVAL(_path) << DVAL(_len));
+        Begin(XSTR() << "FS::Writefile" << DVAL(_path) << DVAL(_len));
```

Comment:
missing space after the method names
```suggestion
        Begin(XSTR() << "FS::Writefile " << DVAL(_path) << DVAL(_len));
```

Reply:
Also there's no need for the `FS::` prefix